releases/latest/edge/bundle.yaml

```diff
@@ -221,8 +216,8 @@ relations:
   - [istio-pilot:istio-pilot, istio-ingressgateway:istio-pilot]
   - [istio-pilot:ingress, tensorboards-web-app:ingress]
   - [istio-pilot:gateway-info, tensorboard-controller:gateway-info]
-  - [katib-db-manager:relational-db, katib-db:database]
-  - [kfp-api:relational-db, kfp-db:database]
+  - [katib-db-manager:relational-db, kf-mysql-db:database]
+  - [kf-mysql-db:database, kfp-api:relational-db]
```

Comment:
Can you swap places?
First our charm, then db?

Reply:
Missed it. Will fix.